webapp/src/main/webapp/iaf/gui/views/Webservices.html

```diff
@@ -33,8 +33,16 @@ <h4>Available ApiListeners</h4>
 					<div class="col-md-12">
 						<table class="table table-striped">
 							<tbody>
+								<tr>
+									<td>0</td>
+									<td>/</td>
+									<td>ALL</td>
+									<td>
+										<a target="_blank" href="{{::rootURL}}api/openapi.json" class="btn btn-xs pull-right default" type="button"><i class="fa fa-arrow-circle-o-down aria-hidden="true"></i> <span>OpenAPI 3.0</span></a>
+									</td>
+								</tr>
```

Comment:
I'm not sure if I like the idea of adding this 'endpoint' to the list, as technically it's not an api service... Maybe we can add a button in the `<div class="ibox-title">` bar or something?